packages/api/test/nfts-upload.spec.js

```diff
@@ -116,13 +120,13 @@ describe('NFT Upload ', () => {
     const { ok, value } = await res.json()
     assert.deepStrictEqual(value.files, [
       { name: 'name1.png', type: 'application/octet-stream' },
-      { name: 'name1.png', type: 'application/octet-stream' },
+      { name: 'name2.png', type: 'application/octet-stream' },
     ])
     assert.ok(value.type === 'directory', 'should be directory')
-    assert.equal(value.size, 71, 'should have correct size')
+    assert.equal(value.size, 138, 'should have correct size')
```

Comment:
Ok so in this test I _did_ change the second file's name (so that I could test manually), but I think these sizes were just wrong. Here's my manual verification:

```console
$ mkdir dir
$ cd dir
$ echo -n 'hello world! 1' > name1.png
$ echo -n 'hello world! 2' > name2.png
$ cd ../
$ ipfs add --cid-version 1 -r dir
added bafkreidsnixyep54glvcz2ocszbokylqalkio2eintcio5tix2vrbmaatu dir/name1.png
added bafkreidekh6xmx5iqumo63i2fipsdtjmpzj4liok7wzptv4tvmnj2ptu6u dir/name2.png
added bafybeieoib3k2cy2x7nlmsfpid3fhnhlewib7ekatui4i27qfio6vutmzm dir
 28 B / 28 B [===========================================================================================================] 100.00%
$ ipfs object stat bafybeieoib3k2cy2x7nlmsfpid3fhnhlewib7ekatui4i27qfio6vutmzm
NumLinks:       2
BlockSize:      110
LinksSize:      108
DataSize:       2
CumulativeSize: 138
```